pinot-plugins/pinot-input-format/pinot-json/src/main/java/org/apache/pinot/plugin/inputformat/json/JSONRecordReader.java

```diff
@@ -86,12 +91,14 @@ public GenericRow next(GenericRow reuse) {
   public void rewind()
       throws IOException {
     _iterator.close();
+    _inputStream.close();
```

Comment:
do you need to rewind the stream?

Reply:
The `init()` should now take care of re-initializing the stream.